compat-data.schema.json

```diff
@@ -1,72 +1,124 @@
-        "stable": { "type": ["string", "boolean"] },
+        "standard_track": { "type": ["string", "boolean"] },
         "obsolete": { "type": ["string", "boolean"] }
       },
-      "required": ["experimental", "standardized", "stable", "obsolete"],
+      "required": ["experimental", "standard_track", "obsolete"],
       "additionalProperties": false
     },
 
-    "feature": {
+    "subfeature": {
       "type": "object",
       "properties": {
-        "Android": { "$ref": "#/definitions/support_statement" },
-        "Android Webview": { "$ref": "#/definitions/support_statement" },
-        "Chrome": { "$ref": "#/definitions/support_statement" },
-        "Chrome for Android": { "$ref": "#/definitions/support_statement" },
-        "Edge": { "$ref": "#/definitions/support_statement" },
-        "Edge Mobile": { "$ref": "#/definitions/support_statement" },
-        "Firefox": { "$ref": "#/definitions/support_statement" },
-        "Firefox for Android": { "$ref": "#/definitions/support_statement" },
-        "IE Mobile": { "$ref": "#/definitions/support_statement" },
-        "Internet Explorer": { "$ref": "#/definitions/support_statement" },
-        "Node.js": { "$ref": "#/definitions/support_statement" },
-        "Opera": { "$ref": "#/definitions/support_statement" },
-        "Opera Mobile": { "$ref": "#/definitions/support_statement" },
-        "Safari": { "$ref": "#/definitions/support_statement" },
-        "Safari Mobile": { "$ref": "#/definitions/support_statement" },
-        "Servo": { "$ref": "#/definitions/support_statement" },
+        "desc": { "type": ["string"] },
+        "support": { "$ref": "#/definitions/compat_block" },
         "status": { "$ref": "#/definitions/status_statement" }
+      }
+    },
+
+    "compat_block": {
+      "type": "object",
+      "properties": {
+        "webview_android": { "$ref": "#/definitions/support_statement" },
+        "chrome": { "$ref": "#/definitions/support_statement" },
+        "chrome_android": { "$ref": "#/definitions/support_statement" },
+        "edge": { "$ref": "#/definitions/support_statement" },
+        "edge_mobile": { "$ref": "#/definitions/support_statement" },
+        "firefox": { "$ref": "#/definitions/support_statement" },
+        "firefox_android": { "$ref": "#/definitions/support_statement" },
+        "ie_mobile": { "$ref": "#/definitions/support_statement" },
+        "ie": { "$ref": "#/definitions/support_statement" },
+        "opera": { "$ref": "#/definitions/support_statement" },
+        "opera_android": { "$ref": "#/definitions/support_statement" },
+        "safari": { "$ref": "#/definitions/support_statement" },
+        "safari_ios": { "$ref": "#/definitions/support_statement" },
+        "servo": { "$ref": "#/definitions/support_statement" }
       },
-      "required": ["Chrome", "Opera", "Edge", "Firefox", "Firefox for Android"],
-      "additionalProperties": false
+      "additionalProperties": true
```

Comment:
Why \`true\`?
(I feel like being restrictive now makes it easier for us to be less restrictive in future without breaking compat.)